AI/pipelines/daily_routine.py

```
@@ -19,93 +37,186 @@
     except Exception as e:
-        print(f"❌ 모델 초기화 실패: {e}")
+        print(f"❌ 모델/스케일러 초기화 실패: {e}")
         return
 
     # XAI 초기화
     xai_generator = None
     if enable_xai:
         try:
-            xai_generator = ReportGenerator(use_gpu_llm=True)
+            xai_generator = ReportGenerator(use_api_llm=False) #true로 하면 GroqClient 사용 (api 토큰 사용), False로 하면 OllamaClient 사용 (로컬 CPU)
         except:
```

Ruff (0.15.2)

[error] Do not use bare `except`

(E722)